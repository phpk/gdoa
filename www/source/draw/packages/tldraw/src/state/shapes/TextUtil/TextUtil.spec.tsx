import { Text } from '..'

describe('Text shape', () => {
  it('Creates a shape', () => {
    expect(Text.create({ id: 'text' })).toMatchSnapshot('text')
  })
})
