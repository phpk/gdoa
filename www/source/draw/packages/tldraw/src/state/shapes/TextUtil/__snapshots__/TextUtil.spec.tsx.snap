// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text shape Creates a shape: text 1`] = `
Object {
  "childIndex": 1,
  "id": "text",
  "name": "Text",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "style": Object {
    "color": "black",
    "dash": "draw",
    "font": "script",
    "isFilled": false,
    "scale": 1,
    "size": "small",
    "textAlign": "middle",
  },
  "text": " ",
  "type": "text",
}
`;
